monitor/grafana/dashboards/operate.json

```diff
@@ -631,26 +646,22 @@
           "refId": "A"
         }
       ],
-      "timeFrom": null,
-      "timeShift": null,
       "title": "Number of records per type",
       "type": "stat"
     }
   ],
   "refresh": "10s",
-  "schemaVersion": 27,
+  "schemaVersion": 38,
   "style": "dark",
   "tags": [],
   "templating": {
     "list": [
       {
         "current": {
           "selected": false,
-          "text": "Prometheus",
-          "value": "Prometheus"
+          "text": "Thanos: production-worker-1",
```

Comment:
@than-pet wait not sure whether this will work in general. This is more specific to our SaaS setup.

Reply:
ok let me fix this as follow up 😁

Reply:
ah indeed, we could revert this block, as it's not making any difference